gallery/tutorials/tutorials/micrograph_source.py

```diff
@@ -0,0 +1,220 @@
+"""
+=================
+Micrograph Source
+=================
+
+This tutorial will demonstrate how to set up and use ASPIRE's ``MicrographSource`` class.
+"""
+
+import numpy as np
+
+from aspire.image import Image
+from aspire.noise import WhiteNoiseAdder
+from aspire.operators import RadialCTFFilter
+from aspire.source import MicrographSource, Simulation
+
+# %%
+# Creating a Micrograph Source
+# ----------------------------
+# We need to generate our projections by creating a ``Simulation`` and passing it into our MicrographSource
+
+# Let's create our Simulation with a particle box size of 64 and one volume.
+n_particles_per_micrograph = 4
+n_micrographs = 4
+
+sim = Simulation(
+    L=64,
+    n=n_particles_per_micrograph * n_micrographs,
+    seed=1234,
+    C=1,
+    amplitudes=1,
+    offsets=0,
+)
+
+# %%
+# We'll pass our Simulation as an argument and give our MicrographSource other arguments.
+# In this example, our MicrographSource has 4 micrographs of size 500, each with 4 particles.
+micrograph = MicrographSource(
+    sim,
+    particles_per_micrograph=n_particles_per_micrograph,
+    micrograph_size=500,
+    micrograph_count=n_micrographs,
+    seed=1234,
+)
+
+# Plot the Micrographs
+micrograph.micrographs[:].show()
+
+# %%
+# CTF Filters
+# -----------
+# By default, no CTF corruption is configured. To apply CTF filters, we have to pass them as arguments to the Simulation
+
+# Create our list of CTF filters
+ctfs = [
+    RadialCTFFilter(pixel_size=4, voltage=200, defocus=15000, Cs=2.26, alpha=0.07, B=0),
+    RadialCTFFilter(pixel_size=4, voltage=200, defocus=10000, Cs=2.26, alpha=0.07, B=0),
+    RadialCTFFilter(pixel_size=4, voltage=200, defocus=20000, Cs=2.26, alpha=0.07, B=0),
+]
+
+# Pass the CTFs into the Simulation, and create a MicrographSource using the same arguments as before
+corrupted_sim = Simulation(
+    L=64,
+    n=n_particles_per_micrograph * n_micrographs,
+    C=1,
+    amplitudes=1,
+    offsets=0,
+    unique_filters=ctfs,
+    seed=1234,
+)
+corrupted_micrograph = MicrographSource(
```

Comment:
I think it's fine to keep the same variable name, since we're not using the original sources farther down.